install/dojo/install.yml

```diff
@@ -0,0 +1,13 @@
+id: dojo
+name: Dojo
+title: Dojo monitoring
+description: |
+  With New Relic One's Dojo monitoring solution, you get full visibility into the complete webpage life cycle of your application or website.
```

Comment:
We no longer use "New Relic One" to refer to our platform. In future, simply use "New Relic".

I'd rewrite this sentence as:

"With New Relic, monitor your Dojo solution to get full visibility into the core web vitals performance of your application or website."